include/deal.II/hp/fe_collection.h

```diff
@@ -505,6 +505,26 @@ namespace hp
     void
     set_default_hierarchy();
 
+    /**
+     * Returns a sequence of FE indices that corresponds to the registered
+     * hierarchy in ascending order, i.e., FE indices are sorted from lowest to
+     * highest level.
+     *
+     * Multiple sequences of FE indices are possible with a single custom
+     * hierarchy, which can be registered with set_hierarchy(). This function
+     * will return the sequence that contains the user-provided index
+     * @p fe_index. The default hierarchy consists of only one sequence.
```

Comment:
I suspect that by the "default" case you mean the one that corresponds to the defaulted argument? Doesn't that correspond to the element with index zero?

In any case, I don't quite understand what that default case would correspond to. What would the sequence be if you had a collection of `FE_Q(2)`, `FE_DGQ(0)`, and `FE_DGPMonomial(1)`?

Reply:
I've been more elaborate on the latest commit.

The default hierarchy corresponds to the FE indices in ascending order and corresponds to the most basic use case, i.e., `FE_Q` elements with degrees in ascending order.

For any wild collection of finite elements, i.e., `FE_Q(2), FE_DGQ(0), FE_DGPMonomial(1)`, the user needs to provide a custom hierarchy. This hierarchy can correspond to different sequences of finite elements, e.g., continuous and discontinuous elements.

For an `FECollection` `FE_Q(1), FE_Q(2), FE_DGQ(1), FE_DGQ(2)`, such a hierarchy might contain two sequences `0,1` for continous elements and `2,3` for discontinous ones. However, we need to know which one of them we want returned, hence the provided parameter of the function.